src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public static final int MAX_VALUE_OF_PARAMETER = 25;
+
+    public static final int DEFAULT_RADIUS = 10;
+
+    public static final String DEFAULT_COLOR = "White";

```

Comment:
It is necessary to use enam
```suggestion
    public static final Color DEFAULT_COLOR = Color.WHITE;
```